### DISCUSSION TITLE
Get files in media group by media_group_id

### DISCUSSION THREAD

**Discussion by Elendiar** [python-telegram-bot/python-telegram-bot#3143] [Q&A] [answered]
Hi, how can i get multiple images sended by user? With single file its easy, but when multiple sended, bot still get one **photo**  with **media_group_id**.
So how i can **get all** other files in media group, to **download** and **process** them? Unfortunately I didn`t see the methods in the documentation. Thanks.

**Comment by Poolitzer** [comment, marked as answer]
You cant get them, you get an update for each media. Doc: https://github.com/python-telegram-bot/python-telegram-bot/wiki/Frequently-requested-design-patterns#how-do-i-deal-with-a-media-group

**Reply by own-mind** [reply]
This is a very inconvenient solution. I think it would be easier if you could at least know the size of the group.

**Reply by Poolitzer** [reply]
Not something we can change, feel free to create an issue at the bot API repo.

**Reply by sviatoslavadamiv** [reply]
Hi, everyone! I have also encountered this problem. A user sends a bot a group of images and I am unable to grab them. Since the bot only posts the last picture. If you could explain how I can get all the photos from the group that the user sent me, I would be extremely grateful!

I tried to get the data with get.updates(), but I sometimes get error 409. I do this code and I get the error:

if message.media_group_id:
    updates = get_update_list()
    for update in updates:
        print(update.message)


Error:
2023-10-06 18:53:26,204 (__init__.py:1083 MainThread) ERROR - TeleBot: "Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running"
